src/pages/selectGame/selectGame.js

```diff
@@ -1,28 +1,23 @@
 import React from 'react';
 import HeaderContainer from '../../containers/header';
+import GameSettings from '../../containers/gamesettings/gameSettings';
+import { Button } from '../../components';
 
 import './styles/selectGame.css';
 
 export default function SelectGame() {
   return (
     //functions
-      // header - check
+    // header - check
     //game type singles doubles
     //player types
     //scoretype
     //backbutton
```

Comment:
Do you need this comments anymore? If yes, you should update it!?

Reply:
updated :)